src/ontology/decision-edit.owl

```diff
@@ -3912,6 +4011,82 @@ BUT: diseases have phases - through latent/incubation/contagious/ initial / subs
     
 
 
+    <!-- http://purl.obolibrary.org/obo/RO_067d925e_a7a1_475f_b6a6_adfe20d159810002092 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/RO_067d925e_a7a1_475f_b6a6_adfe20d159810002092">
```

Comment:
This URI needs to be fixed - that Protege setup one must do to get the editor to issue a new ID.  If you need tips on how to get a good ID after the fact, let me know and we can do it together.